ci/trigger_ci.sh

```diff
@@ -5,7 +5,14 @@ mark_failure()
     echo "$1"
     echo "=================================================="
     echo "CI complete releasing the nodes"
-    cico node done $cico_node_key
+    if [ $CI_DEBUG -eq 0]
+    then
+        cico node done $cico_node_key
+    else
+        echo "============================================================"
+        echo "DEBUG mode is set for CI, keeping nodes for debugging"
```

Comment:
you need to define a debug period here, for example sleep for M seconds for debug, after lapsing of M seconds, we need to release the nodes.

This is to ensure the nodes are not left hanging behind if not debugged within specified time.

We had this behavior earlier.